src/frontend/src/pages/CreateChangeRequestPage/CreateChangeRequestView.tsx

```diff
@@ -129,22 +183,27 @@ const CreateChangeRequestsView: React.FC<CreateChangeRequestViewProps> = ({
       }}
     >
       <PageTitle title="New Change Request" previousPages={[{ name: 'Change Requests', route: routes.CHANGE_REQUESTS }]} />
-      <PageBlock title="Details">
+      <PageBlock title="Details" transparentBackground>
         <Grid container spacing={2}>
-          <Grid item xs={12}>
+          <Grid item xs={6}>
             <FormLabel>WBS</FormLabel>
             <NERAutocomplete
               id="wbs-autocomplete"
               onChange={wbsAutocompleteOnChange}
               options={wbsDropdownOptions}
               size="small"
+              sx={{
```

Comment:
why'd you add this

Reply:
so that it would look good with the background